Sources/PrimerSDK/Classes/Data Models/PaymentNetwork.swift

```diff
@@ -0,0 +1,154 @@
+//
+//  PaymentNetwork.swift
+//  PrimerSDK
+//
+//  Created by Evangelos Pittas on 16/4/21.
+//
+
+#if canImport(UIKit)
+
+import Foundation
+import PassKit
+
+public enum PaymentNetwork: String {
+    
+    case chinaUnionPay
+    
+    case discover
+    
+    @available(iOS 12.0, *)
```

Comment:
[Feedback]
These payment networks will also be used in the card form (to filter out specific card networks), and they should be available regardless of the version of iOS.
Not sure we should restrict the availability of these payment networks. Unless you plan to have a different data models for card networks specifically?

Reply:
At the moment this is used only by Apple Pay. You're right though, a more generic model is needed since we can re-use this on the card models.

Fixed.